### ISSUE TITLE
how to convert l2norm to trt

### ISSUE THREAD

**Issue by secretdragon** [NVIDIA-AI-IOT/torch2trt#92]
I am trying to use the torch version of refinedet to use trench2trt for trt acceleration, which has a l2norm need to be converted, there is a problem in the conversion process, I do not know how to solve, if you have a better solution, thank you very much for your help!!!
`class L2Norm(nn.Module):
    def __init__(self,n_channels, scale):
        super(L2Norm,self).__init__()
        self.n_channels = n_channels
        self.gamma = scale or None
        self.eps = 1e-10
        self.weight = nn.Parameter(torch.Tensor(self.n_channels))
        self.reset_parameters()

    def reset_parameters(self):
        init.constant_(self.weight,self.gamma)

    def forward(self, x):
        norm = x.pow(2).sum(dim=1, keepdim=True).sqrt() + self.eps
        x = torch.div(x,norm)
        out = self.weight.unsqueeze(0).unsqueeze(2).unsqueeze(3).expand_as(x) * x
        return out`
error:
![image](https://user-images.githubusercontent.com/35625414/66214899-23973980-e6f5-11e9-9441-cf7f3e360f2d.png)

**Comment by jaybdub** [comment]
Hi secretdragon,

It looks like the unsupported layers here are ``unsqueeze`` and ``expand_as``.  I will consider adding these.

In the meantime, I would guess that rather than relying on atomic converters to do the full conversion (parsing all of the sub-ops), you might benefit from writing your own converter unique to this module.

```python
@tensorrt_converter('L2Norm.forward')  # full path (ie: package.module.class.method)
def converter_l2norm(ctx):
    #  tensorrt python API calls
```

Please see the README for more details, and let me know if you have any questions.

Best,
John

**Comment by secretdragon** [comment]
Thank you very much for your reply, but I am not very good at writing my own converter unique to this module, there are other similar modules in refinedet. I have referenced your opinion which is "detection models will share a backbone architecture which is supported by TensorRT. It may be possible to optimize just the backbone module, and run the rest in Python/PyTorch.",
![image](https://user-images.githubusercontent.com/35625414/66256143-8eac4300-e7bd-11e9-960a-627619c5fb6a.png)
but l donnt know how to do that,Can you introduce related examples to teach me how to do that?Looking forward to your reply.

**Comment by secretdragon** [comment]
> Hi secretdragon,
> 
> It looks like the unsupported layers here are `unsqueeze` and `expand_as`. I will consider adding these.
> 
> In the meantime, I would guess that rather than relying on atomic converters to do the full conversion (parsing all of the sub-ops), you might benefit from writing your own converter unique to this module.
> 
> ```python
> @tensorrt_converter('L2Norm.forward')  # full path (ie: package.module.class.method)
> def converter_l2norm(ctx):
>     #  tensorrt python API calls
> ```
> 
> Please see the README for more details, and let me know if you have any questions.
> 
> Best,
> John

I have solved this problem. Here is my solution
![image](https://user-images.githubusercontent.com/35625414/69524065-39600700-0fa0-11ea-8e99-dbc6b66b5f11.png)
